requirements/base-requirements.in

```diff
@@ -77,7 +77,7 @@ Pillow
 polib
 prometheus-client
 psycogreen
-psycopg2>=2.8.4  # Python 3.8 support
+psycopg2
```

Comment:
I think it's fine, but curious why this was removed? It seems unrelated to and unnecessary for M2 chips.